misc/config_tools/schema/types.xsd

```diff
@@ -151,10 +173,17 @@ Read more about the available scheduling options in :ref:`cpu_sharing`.</xs:docu
   </xs:restriction>
 </xs:simpleType>
 
+<xs:simpleType name="ProviderType">
+  <xs:restriction base="xs:string">
+    <xs:enumeration value="Hypervisor" />
+    <xs:enumeration value="Device model" />
```

Comment:
Is case significant?  We usually refer to "Device Model" in the documentation.

Reply:
I do this following the style that only the first letter of the first word is capitalized in enumerations. That said, perhaps we can consider "Device Model" as a proper noun and capitalize the first letter of every word.